phf/src/map.rs

```diff
@@ -47,6 +47,16 @@ where
 }
 
 impl<K, V> Map<K, V> {
+    /// Create a new, empty, immutable map.
+    #[inline]
+    pub const fn new_empty() -> Self {
```

Comment:
Bikeshedding: In general, the Rust community names `new` to such a method.

```suggestion
    pub const fn new() -> Self {
```